Known_Issues/All VMMS operations are blocked on Silver or higher durability.md

```diff
@@ -0,0 +1,100 @@
+# VMMS operations are blocked node gets stuck in deletion and creation after deleting a VMSS
+
+## Problem
+- All VMSS operations are blocked on Silver or higher durability Scaleset after trying to delete a node from a VMMS 
+- The node being deleted (seed node) is stuck in Disabling state
```

Comment:
suggestion: expand what seed nodes are (seed node which are used by the Service Fabric cluster services)
or
The node being deleted is a seed node, which are used by the Service Fabric cluster services, and is stuck in Disabling state